contracts/satoshi-bridge/src/api/token_receiver.rs

```diff
@@ -15,6 +15,7 @@ pub enum TokenReceiverMessage {
         output: Vec<TxOut>,
         max_gas_fee: Option<U128>,
         orchard_bundle_bytes: Option<String>,
```

Comment:
We can use [Base64VecU8](https://docs.rs/near-sdk/latest/near_sdk/json_types/struct.Base64VecU8.html) type for this.

Reply:
fixed: https://github.com/Near-One/btc-bridge/pull/20/commits/7abcc1462cda80f8dc1a784ab0d36e163c24f40f